src/usr/local/pfSense/include/www/firewall_nat_1to1.inc

```diff
@@ -229,6 +229,11 @@ function save1to1NATrule($post, $id, $json = false) {
 
 		if (write_config(gettext("Firewall: NAT: 1:1 - saved/edited NAT 1:1 mapping.")) && !$json) {
 			mark_subsystem_dirty('natconf');
+
+			if (isset($config['unbound']['enable']) && isset($config['unbound']['replace1to1'])) {
```

Comment:
This check should use the newer functions like the one in the next part of the diff:

```php
if (config_path_enabled('unbound') && config_path_enabled('unbound', 'replace1to1')) {
```